sqlmesh/dbt/manifest.py

```diff
@@ -136,9 +136,24 @@ def _load_tests(self) -> None:
             if node.resource_type != "test":
                 continue
 
+            skip_test = False
+            package_name = node.package_name
+            refs = _refs(node)
+            for ref in refs:
+                ref_node_name = f"model.{ref}" if "." in ref else f"model.{package_name}.{ref}"
```

Comment:
We need to check for model, seed, and snapshot.

Reply:
Maybe check all nodes that don't start with test?

Reply:
ah, good catch, thanks!